qiskit/algorithms/evolvers/variational/var_qite.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self,
         ansatz: Union[OperatorBase, QuantumCircuit],
         variational_principle: Optional[ImaginaryVariationalPrinciple] = None,
-        ansatz_init_param_values: Optional[
+        initial_parameters: Optional[
```

Comment:
I was wondering why I did not see this change show up in tests and say example above. In these it seems the parameters are always passed positionally.